contracts/SSVNetwork.sol

```diff
@@ -327,31 +532,113 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
         pod.validatorCount += activeValidatorCount;
 
-        if (_liquidatable(pod.usage.balance, pod.validatorCount, newOperatorIds)) {
-            revert AccountLiquidatable();
+        if (_liquidatable(pod.disabled, _ownerPodBalance(pod, podIndex), pod.validatorCount, newOperatorIds)) {
+            revert PodLiquidatable();
         }
     }
 
-
     // TODO add external functions below to interface
 
+    // @dev external dao functions
+
+    function getOperatorFeeIncreaseLimit() external view returns (uint64) {
+        return _operatorMaxFeeIncrease;
+    }
+
+    function getExecuteOperatorFeePeriod() external view returns (uint64) {
+        return _executeOperatorFeePeriod;
+    }
+
+    function getDeclaredOperatorFeePeriod() external view returns (uint64) {
+        return _declareOperatorFeePeriod;
+    }
+
+    function getNetworkFee() external view returns (uint256) {
+        return _networkFee.expand();
+    }
+
+    function getNetworkBalance() external onlyOwner view returns (uint256) {
+        DAO memory dao = _dao;
+        return _networkBalance(dao).expand();
+    }
+
+    function updateOperatorFeeIncreaseLimit(uint64 newOperatorMaxFeeIncrease) external onlyOwner {
+        _updateOperatorFeeIncreaseLimit(newOperatorMaxFeeIncrease);
+    }
+
+    function updateDeclareOperatorFeePeriod(uint64 newDeclareOperatorFeePeriod) external onlyOwner {
+        _updateDeclareOperatorFeePeriod(newDeclareOperatorFeePeriod);
+    }
+
+    function updateExecuteOperatorFeePeriod(uint64 newExecuteOperatorFeePeriod) external onlyOwner {
+        _updateExecuteOperatorFeePeriod(newExecuteOperatorFeePeriod);
+    }
+
+    function updateNetworkFee(uint256 fee) external onlyOwner {
+        DAO memory dao = _dao;
+        dao = _updateDAOEarnings(dao);
+        _dao = dao;
+
+        _updateNetworkFeeIndex();
+
+        emit NetworkFeeUpdate(_networkFee.expand(), fee);
+
+        _networkFee = fee.shrink();
+    }
+
+    function withdrawDAOEarnings(uint256 amount) external onlyOwner {
+        DAO memory dao = _dao;
+
+        if(amount.shrink() > _networkBalance(dao)) {
+            revert NotEnoughBalance();
+        }
+
+        dao.withdrawn += amount.shrink();
+        _dao = dao;
+
+        _token.transfer(msg.sender, amount);
+
+        emit NetworkFeesWithdrawal(amount, msg.sender);
+    }
+
     // @dev external operators functions
 
-    function operatorSnapshot(uint64 id) external view returns (uint64 currentBlock, uint64 index, uint64 balance) {
+    function operatorSnapshot(uint64 id) external view returns (uint64 currentBlock, uint64 index, uint256 balance) {
         Snapshot memory s = _getSnapshot(_operators[id], uint64(block.number));
-        return (s.block, s.index, s.balance);
+        return (s.block, s.index, s.balance.expand());
+    }
+
+    // @dev internal dao functions
+
+    function _updateOperatorFeeIncreaseLimit(uint64 newOperatorMaxFeeIncrease) private {
```

Comment:
@adamzigdon @vadiminc is it really best practice to have for each function private and public